qualtran/testing.py

```diff
@@ -236,9 +235,7 @@ def assert_wire_symbols_match_expected(bloq: Bloq, expected_ws: List[str]):
     ws = []
     regs = bloq.signature
     for i, r in enumerate(regs):
-        # note this will only work if shape = ().
-        # See: https://github.com/quantumlib/Qualtran/issues/608
-        ws.append(bloq.wire_symbol(Soquet(i, r)).text)
+        ws.append(bloq.wire_symbol(r, (i,)).text)
```

Comment:
I don't think this fixes the linked issue. Can you restore the comment

Reply:
Done.  If you have a good example, I can fix the comment too.

Reply:
how about `Toffoli()`